setup.cfg

```diff
@@ -56,8 +57,7 @@ tests =
     pytest==6.2.5
 
 [options.package_data]
-* = emmo_lib/emmo_units.csv,
-    emmo_lib/emmo_unit_prefixes.csv
+* = *.csv
```

Comment:
shouldn't it be `emmo_lib/*.csv`? or does the asterisk include all file in the subdirectories?

Reply:
I tried it make simple, just to make it work. I think there are only these two csv files in the entire repo.